EKF/terrain_estimator.cpp

```diff
@@ -80,7 +91,7 @@ void Ekf::runTerrainEstimator()
 	checkRangeDataContinuity();
 
 	// Perform initialisation check
```

Comment:
Suggest we add a comment to indicate that continually re-initializing on the ground is deliberate.